ramls/marccat.raml

```diff
@@ -98,3 +99,29 @@ resourceTypes:
               description: "Return a list of marc records in text format"
             500:
               description: "Internal server error"
+
+  /searchAuth:
+    displayName: Search authority marc records
+    type:
+      collection-item:
+        exampleItem: !include examples/searchAuthResult.json
+        schema: searchResult
+    get:
+      is: [pageable,
+          searchable: {description: "using CCL query in q parameter",
```

Comment:
Pardon my ignorance - is this supposed to be CQL, or is there something called CCL I'm just not aware of?

Reply:
CCL is the query for our Search engine.